components/DataPage/BabydegenMetrics.jsx

```diff
@@ -87,7 +126,41 @@ export const BabydegenMetricsInfo = () => {
             <span>Unavailable</span>
           )}
         </p>
-        <CodeSnippet>{dailyStakingGlobalsSnapshotsQuery}</CodeSnippet>
+        <CodeSnippet>{OPTIMISM_STAKING_QUERY_STRING}</CodeSnippet>
+
+        <h3 className={`${TEXT_MEDIUM_CLASS} font-bold mt-10`}>
+          Modius Population Metrics Query
+        </h3>
+        <p className="text-purple-600">
+          Subgraph link{' '}
+          {babydegenModeSubgraphLinks.length > 0 ? (
```

Comment:
there's only one link, and there can't be more. can we remove this logic, it makes reading unnecessary complex